digital-objects/asct-b/allen-brain/draft/raw/metadata copy.yaml

```diff
@@ -0,0 +1,64 @@
+title: Anatomical Structures, Cell Types, plus Biomarkers (ASCT+B) table for Brain
+description: >-
+  [Brain
+  v1.5](https://cdn.humanatlas.io/hra-releases/v2.0/asct-b/asct-b-allen-brain.csv)
+  is a minor update of the table that provides parent cell type mappings to Cell
+  Ontology for the specific cell types that were added to [Brain
+  v1.4](https://cdn.humanatlas.io/hra-releases/v1.4/asct-b/asct-b-allen-brain.csv),
+  but do not yet have Cell Ontology (CL) or Provisional Cell Ontology (PCL) IDs.
+  More of the brain anatomy was also mapped to the Uberon multispecies anatomy
+  ontology based on exact matching synonym terms.
+creators:
+  - fullName: Jeremy A. Miller
+    firstName: Jeremy
+    lastName: Miller
+    orcid: 0000-0003-4549-588X
+  - fullName: Kimberly Siletti
+    firstName: Kimberly
+    lastName: Siletti
+    orcid: 0000-0001-7620-8973
+  - fullName: Richard H. Scheuermann
+    firstName: Richard
+    lastName: Scheuermann
+    orcid: 0000-0003-1355-892X
+project_leads:
+  - fullName: Katy Börner
```

Comment:
Replace o

Reply:
It looks correct in the metadata file as the Börner, are you saying you want Borner instead? That's not how any of the others are written.
@katyb